packages/grafana-ui/src/components/Table/TableCellInspector.tsx

```diff
@@ -28,6 +28,7 @@ export function TableCellInspector({ value, onDismiss, mode }: TableCellInspecto
     if (trimmedValue[0] === '{' || trimmedValue[0] === '[' || mode === 'code') {
```

Comment:
this works for `code` but I'm not so sure about the other 2 - if not, we might need a separate check. @codeincarnate could you please check? Ty!

Reply:
Those are so it won't activate automatically on numeric values but just on JSON array and objects. Not sure if it's something we want to keep, something that I carried forward from the old auto-detect logic.